kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml

```diff
@@ -136,4 +136,7 @@ createDomainScriptName: create-domain-job.sh
 createDomainFilesDir: wlst
 
 # istioEnabled specifies whether or not it is deployed under Istio.
-istioEnabled: false
\ No newline at end of file
+istioEnabled: true
```

Comment:
istio should not be enabled by default in the sample

Reply:
fixed.